src/util/requests/impl/SslContext.cpp

```diff
@@ -82,9 +82,9 @@ getRootCertificate()
 }  // namespace
 
 std::expected<boost::asio::ssl::context, RequestError>
-makeSslContext()
+makeClientSslContext()
 {
-    ssl::context context{ssl::context::sslv23_client};
+    ssl::context context{ssl::context::tls_client};
```

Comment:
Seems unrelated to main refactoring.. just wanted to confirm that this is tested and works fine?

Reply:
I checked, works fine.